starter-code/main.js

```diff
@@ -0,0 +1,72 @@
+Vue.component("message-diplay", {
+  props: ["header", "body", "color"],
+  data() {
+    return { show: true };
+  },
+  template: `
+  <article v-show="show" :class='["message", "is-"+color]'>
+    <div class="message-header">
+      <p>{{header}}</p>
+      <button @click="show =!show" class="delete" aria-label="delete"></button>
+    </div>
+    <div class="message-body">
+    {{body}}
+    </div>
+  </article>
+  `
+});
+
+Vue.component("message-country", {
+  props: ["country", "capital", "color"],
+  data() {
+    return { show: true };
+  },
+  template: `
+  <article v-show="show" :class='["message", "is-"+color]'>
```

Comment:
If you wanted to display a message for each country you could have reused the `message-display` component instead.